configuration/persistence.md

```diff
@@ -7,12 +7,238 @@ title: Persistence
+Persistence may also be used to store values that are subsequently displayed in a graph.
+(For an excellent graphing tutorial see [InfluxDB+Grafana persistence and graphing](https://community.openhab.org/t/influxdb-grafana-persistence-and-graphing/13761).
 
-{% include contribution-wanted.html %}
+openHAB persists item states in a database, and most popular databases are supported.
+You may have more that one persistence add-on loaded, and each of these may be configured independently.
+Examples of available persistence add-ons include relational databases, NoSQL databases, round-robin databases, Internet-of-Things (IoT) cloud services, simple log files, etc.
+All of these options are available in openHAB, and they are all configured in the same way.
+Note that the type of database(s) you use for persistence may be influenced by your requirements for exporting data (e.g. IoT services or log files), or your requirement to make queries against the database to retrieve historical data (e.g. MySQL and MongoDB).
+A complete list of supported persistence add-ons may be found in the [persistence]({{base}}/addons/persistence.html) section of the on-line openHAB documentation.
+
+## Persistence Add-on Configuration
+
+Each persistence add-on you install will need to be configured.
+Please refer to the specific [on-line documentation]({{base}}/addons/persistence.html) for your selected persistence add-on for configuration instructions.
+
+## Default Persistence Service
+
+You may install more than one persistence add-on.
+Therefore, it is important to select a default persistence service.
+You should do this even if you have only one persistence add-on installed.
+
+To select a default persistence service, in paperUI, select Configuration and then System from the side menu.
+Scroll down to "Persistence", and select your Default Service from the drop-down list.
+Note that you must first install a persistence add-on before you make this selection.
+Be sure to save your choice once you have selected your default service.
+
+## Configuration
+
+You can choose how your persistence service operates, which item states it persists, and under what conditions it persists them.
+These things and more are configured in a file named `<persistenceservice>.persist`, where "persistenceservice" is replaced by the name of your add-on (e.g. `rrd4j.persist`).
+This folder is located in `$OPENHAB_CONF/persistence`.
+The configuration defines persistence "strategies" which are very similar to [triggers]({{base}}/configuration/rules-dsl.html#rule-triggers) in openHAB rules.
+These strategies may be used to persist an item state when some bus event has occurred (e.g. an item state has been updated or changed), or on a schedule or at a specific time of day (e.g. through a [cron expression](http://www.quartz-scheduler.org/documentation/quartz-2.1.x/tutorials/crontrigger).
+
+Persistence configuration files consist of several sections:
```

Comment:
Instead of the two code blocks in the below sections how about one less abstract example here.
In the Items and the Transformations article I've switched over to simple examples rather than non intuitive formal definitions. We should strive for a good middle path...

Reply:
Yep - the point is to improve comprehension, not prove how clever one can be.  People can go to the Community and tutorials for more complex examples.  Generally speaking, it is my feeling that this documentation should aim to get people up and running with a basic system.